### PR TITLE
designing sign up

### PR DESCRIPTION
**Pull request by suryadas-trenser**

### CODE REVIEW

File: WEATHERAPP/Components/SignUp/SignUp.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+   background-image:url("../../Assets/L&D-BG\ 4.png");
+   display: flex;
+   flex-direction: column;
+   justify-content: space-evenly;
+   align-items: center;
+}
+.app-header
+{
+   text-align: center;
+   font:700 72px system-ui;
```
Comment (AkhilTrenser-123) [inline WEATHERAPP/Components/SignUp/SignUp.css:17]: Replace with font-weight: 700; font-size: 72px.

File: WEATHERAPP/Components/SignUp/SignUp.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+}
+.signup-form
+{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+    background-color: rgba(132, 95, 95, 1);
```
Comment (AkhilTrenser-123) [inline WEATHERAPP/Components/SignUp/SignUp.css:25]: Move colors to another css file and use using var here

File: WEATHERAPP/Components/SignUp/SignUp.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    .signup-form
+    {
+        width: 200px;
+        height: 225px;
+    }
+    .input-fields
+   {
+        width: 160px;
+        height: 20px;
+        border-radius: 5px;
```
Comment (AkhilTrenser-123) [inline WEATHERAPP/Components/SignUp/SignUp.css:76]: Remove -webkit-, -moz-, -ms-, and -o- prefixes, as modern browsers support border-radius.

File: WEATHERAPP/Components/SignUp/SignUp.css [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        -webkit-border-radius: 5px;
+        -moz-border-radius: 5px;
+        -ms-border-radius: 5px;
+        -o-border-radius: 5px;
+   }
+   .signup-button
+   {
```
Comment (AkhilTrenser-123) [inline WEATHERAPP/Components/SignUp/SignUp.css:83]: Add cursor to button

File: WEATHERAPP/Components/SignUp/SignUp.css [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+   height: 30px;
+   border-radius: 6px;
+}
+.signup-button
+{
+    width: 270px;
+    height: 30px;
+    background-color: rgba(44, 44, 44, 1);
+    color: rgba(255, 255, 255, 1);
+    border-radius:5px ;
```
Comment (AkhilTrenser-123) [inline WEATHERAPP/Components/SignUp/SignUp.css:58]: Use consistent spacing in the entire file

File: WEATHERAPP/Components/SignUp/SignUp.html [hunk 6]
```diff
@@ -1,2 +1,31 @@
 <!DOCTYPE html>
-<head>signup page of the weather app for new user</head>
```
Comment (AkhilTrenser-123) [inline WEATHERAPP/Components/SignUp/SignUp.html:2]: Add event handlers for the components

File: WEATHERAPP/Components/SignUp/SignUp.html [hunk 7]
```diff
@@ -1,2 +1,31 @@
...
+                <section class="input-section">
+                    <label for="password" class="input-label">Password</label>
+                    <input type="password" class="input-fields" name="password" id="password" placeholder="Password" aria-label="password" required>
+                </section>
+                <section class="input-section">
+                    <label for="email" class="input-label">Email</label>
+                    <input type="email" class="input-fields" name="email" id="email" placeholder="Email" aria-label="email" required>
+                </section>
+                <section class="buttons-section">
+                    <button class="signup-button" id="sign-up">Sign UP</button>
```
Comment (AkhilTrenser-123) [inline WEATHERAPP/Components/SignUp/SignUp.html]: Specify type attribute for button (e.g., type="submit").

File: WEATHERAPP/Components/SignUp/SignUp.html [hunk 8]
```diff
@@ -1,2 +1,31 @@
...
+<html>
+    <head lang="en">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta charset="UTF-8">
+        <title>WEATHER APP</title>
+    <link rel="stylesheet" href="SignUp.css">
+    </head>
+    <body>
+        <div class="background">
+            <header class="app-header">WEATHER APP</header>
```
Comment (AkhilTrenser-123) [inline WEATHERAPP/Components/SignUp/SignUp.html]: Wrap input fields and button in a form element for better semantic structure.

File: WEATHERAPP/Components/SignUp/SignUp.css (hunk 2 above)
Comment (suryadas-trenser) [reply WEATHERAPP/Components/SignUp/SignUp.css:25]: Resolved

File: WEATHERAPP/Components/SignUp/SignUp.css (hunk 3 above)
Comment (suryadas-trenser) [reply WEATHERAPP/Components/SignUp/SignUp.css:76]: Resolved

File: WEATHERAPP/Components/SignUp/SignUp.css (hunk 4 above)
Comment (suryadas-trenser) [reply WEATHERAPP/Components/SignUp/SignUp.css:83]: Resolved

File: WEATHERAPP/Components/SignUp/SignUp.css (hunk 5 above)
Comment (suryadas-trenser) [reply WEATHERAPP/Components/SignUp/SignUp.css:58]: Resolved

File: WEATHERAPP/Components/SignUp/SignUp.html (hunk 6 above)
Comment (suryadas-trenser) [reply WEATHERAPP/Components/SignUp/SignUp.html:2]: Resolved

File: WEATHERAPP/Components/SignUp/SignUp.html (hunk 7 above)
Comment (suryadas-trenser) [reply WEATHERAPP/Components/SignUp/SignUp.html]: Resolved

File: WEATHERAPP/Components/SignUp/SignUp.html (hunk 8 above)
Comment (suryadas-trenser) [reply WEATHERAPP/Components/SignUp/SignUp.html]: Resolved

**Review by AkhilTrenser-123 — APPROVED** [review]

**Pull request merged**